specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,63 @@
+# Charge Subcategory
+
+Charge Subcategory indicates what kind of usage or adjustment the row represents. Charge Subcategory is a supplemental detail to the Charge Type. It provide additional context to describe the primary category of charge.
```

Comment:
Yes, we're waiting to do that after all the columns are in because several columns have references to ChargeType and we want to wait for everything to be in before we rename it to ensure nothing gets missed.